src/Component/Console/src/Event/ConsoleErrorEvent.js

```diff
@@ -8,12 +8,7 @@ const InvalidArgumentException = Jymfony.Component.Console.Exception.InvalidArgu
  */
 class ConsoleErrorEvent extends ConsoleEvent {
     /**
-     * Constructor
-     *
-     * @param {Jymfony.Component.Console.Input.InputInterface} input
-     * @param {Jymfony.Component.Console.Output.OutputInterface} output
-     * @param {Error} error
-     * @param {Jymfony.Component.Console.Command.Command} command
+     * @inheritdoc
```

Comment:
Parent documentation does not cover all the function parameters.
We should document `error` parameter also or revert this modification.